CODEOWNERS

```diff
@@ -18,110 +20,111 @@
 # @cilium/policy        Policy behaviour
 # @cilium/proxy         L7 proxy, Envoy
 # @cilium/vendor        Vendoring, dependency management
-#
-# Specific code owners:
-# @eloycoto             Debian packaging
-# @aanm @ianvernon      Docker packaging
-# @scanf @borkmann      BPF documentation
 
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/janitors
-api/ @cilium/api
-bpf/ @cilium/bpf
-bugtool/cmd/ @cilium/cli
-cilium/ @cilium/cli
-cilium/cmd/preflight_k8s_valid_cnp.go @cilium/kubernetes
-cilium-health/cmd/ @cilium/cli
-cilium-health/launch/ @cilium/health
-contrib/packaging/deb/ @eloycoto
-contrib/packaging/docker/ @aanm @ianvernon
-contrib/vagrant @cilium/ci
-daemon/ @cilium/agent
-daemon/cmd/datapath.* @cilium/bpf
-daemon/cmd/endpoint.* @cilium/endpoint
-daemon/cmd/health.* @cilium/health
-daemon/cmd/hubble.go @cilium/hubble
-daemon/cmd/ipcache.* @cilium/ipcache
-daemon/cmd/loadbalancer.* @cilium/loadbalancer
-daemon/cmd/metrics.* @cilium/metrics
-daemon/cmd/policy.* @cilium/policy
-daemon/cmd/prefilter.go @cilium/bpf
-daemon/cmd/proxy.go @cilium/proxy
-daemon/cmd/state.go @cilium/endpoint
-daemon/cmd/sysctl_linux.go @cilium/bpf
-Documentation/ @cilium/docs
-Documentation/bpf.rst @scanf @borkmann
-Documentation/contributing/ @cilium/contributing
-envoy/ @cilium/proxy
-examples/ @cilium/docs
-examples/kubernetes/ @cilium/kubernetes
-examples/minikube/ @cilium/kubernetes
+/api/ @cilium/api
+/bpf/ @cilium/bpf
+Makefile @cilium/build
```

Comment:
I feel like there was a discussion about this somewhere, but putting this so early means that other Makefiles won't be part of `cilium/build` right? Because the directory listings below this line will take precedence.

This was why we had it at the end of the list previously.

Reply:
(btw, does this cover stuff like `Makefile.defs` as well?)

Reply:
> I feel like there was a discussion about this somewhere, but putting this so early means that other Makefiles won't be part of `cilium/build` right? Because the directory listings below this line will take precedence.

Yes. This is on purpose though. I expect `test/Makefile` to be under `@cilium/ci` and `bpf/Makefile` under `@cilium/bpf`. Maybe we need to extend the `CODEOWNERS` file further to have e.g., `bpf/Makefile` under both `@cilium/bpf` and `@cilium/build`?

I've updated this entry to also match `Makefile.quiet`, `Makefile.defs`, etc. Same for `Dockerfile*`.